llvm/include/llvm/ProfileData/MemProf.h

```diff
@@ -1028,6 +1036,10 @@ class CallStackRadixTreeBuilder {
   getCallStackPos() const {
```

Comment:
Can we remove getCallStackPos and replace its users with takeCallStackPos? Looks like existing callers are all unit tests where we could do this.